app/common/auth/authentication-interceptor.config.js

```diff
@@ -165,10 +149,29 @@ function AuthInterceptor($rootScope, $injector, $q, CONST, Session, _) {
                 $rootScope.$broadcast('event:forbidden');
             }
             deferred.reject(rejection);
-        // For anything else, just forward the rejection
+            // For anything else, just forward the rejection
         } else {
             deferred.reject(rejection);
         }
         return deferred.promise;
     }
+
+    /**
+     * Returns true if url is ignorable, false if not
+     * @param config
+     */
+    function shouldIgnoreAuthError(config) {
```

Comment:
Ironically, this makes the code better :D

We need to get to a new framework. I don't see anything obvious wrong with this but it is probably fragile and should be replaced sooner rather than later.